src/components/LinkButton/LinkButton.jsx

```diff
@@ -1,14 +1,10 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './LinkButton.css';
 
-class LinkButton extends Component {
-  render() {
-    return (
-      <a className="LinkButton" href="https://github.com/firstcontributions/first-contributions/blob/master/README.md">
-        <span> Get started </span>
-      </a>
-    );
-  }
-}
-
-export default LinkButton;
+export default function LinkButton() {
```

Comment:
A second ago I thought changing the LinkButton from class component to functional component it would be way efficient, but you've broken it to terms , saying it would be non uniform. I'm learning a lot here. Any other contributing repositories you've got?